### ISSUE TITLE
Task: Add an authentication header

### ISSUE THREAD

**Issue by ReinderVosDeWael** [childmindresearch/ctk-frontend#5]
### Description

The frontend should authenticate on every request. This is easiest to do through a header, but we'll have to see what's possible with CMI's LDAP. 

- This task is blocked by https://github.com/orgs/cmi-dair/projects/3/views/3?pane=issue&itemId=38407437.

### Tasks

- [ ] Add a authentication header on every request. 

### Freeform Notes

This task is currently pre-mature and should remain on hold until the project has progressed further.

**Comment by ReinderVosDeWael** [comment]
Can be taken care of with Azure SSO
